docs/calicoctl/node.md

```diff
@@ -0,0 +1,211 @@
+
+# User guide for 'calicoctl node' commands
+
+This sections describes the `calicoctl node` commands.
+
+The `calicoctl node` command starts the calico/node Docker image that 
+runs the main Calico processes such as Felix and the BIRD BGP routing 
+daemon.  The calico/node container is a required component for Calico 
```

Comment:
"The calioc/node container is required to be running on every compute host for Calico networking."